demos/uploader/index.html

```diff
@@ -52,11 +52,12 @@
     </head>
     <body>
         <div class="flc-overviewPanel fl-overviewPanel-container"></div>
-        <!-- Basic upload controls, used when JavaScript is unavailable -->
+        <!-- Basic upload controls, used when JavaScript/HTML5 is unavailable -->
         <form method="post" enctype="multipart/form-data" class="fl-progEnhance-basic">
             <p>Use the Browse button to add a file, and the Save button to upload it.</p>
             <input name="fileData" type="file" />
             <input class="fl-uploader-basic-save" type="submit" value="Save"/>
+            <p class="fl-uploader-no-js-note">NOTE : You are using an outdated version of browser. Please upgrade your browser which supports HTML5 <noscript>and JavaScript </noscript>features.</p>
```

Comment:
True, that did come across my mind, but then, wouldn't it be simply better if the user _does_ upgrade it for once and all? :sweat_smile: Let me know what you think